pkg/machinery/go.mod

```diff
@@ -21,6 +21,7 @@ require (
 	github.com/talos-systems/crypto v0.3.5
 	github.com/talos-systems/go-blockdevice v0.3.2-0.20220405134045-d9c3a2738861
 	github.com/talos-systems/go-debug v0.2.1
+	github.com/talos-systems/go-smbios v0.2.0
```

Comment:
I believe it's not used in `machinery`, so we don't need it here

Reply:
yup, it was from the new files i added, not part of this commit